web/app/charts/MapChart.js

```diff
@@ -266,9 +318,28 @@ bluewave.charts.MapChart = function(parent, config) {
         }
         else if (mapLevel === "states"){
 
-            var projection = d3.geoAlbers().center([-8, 43]);
+            var centerLon, centerLat;
+            if(chartConfig.newCenter){
```

Comment:
Can we just call this "center" or "mapCenter" instead of "newCenter"?

Reply:
yes we can.